canonical-web-and-design/www.ubuntu.com
# Moving uasd to live, archive old

## Done

- moved temp UASD page to live and archived older version

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/uasd](http://0.0.0.0:8001/uasd)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- See that it's the [new version](https://docs.google.com/document/d/1vQNdVmpsB3iQP7SnJHxT3A8rbD5jSAzTEoYUOepR-SM/edit)
